docs/components/lists.md

```diff
@@ -0,0 +1,452 @@
+        value: false,
+        onChanged: (value) {
+          // Update value.
+        },
+      ),
+    ),
+  ],
+)
+```
+
+### Three-line list key properties
+
+The following are tables of the list item contents:
+
+
+##### Supporting visuals
+
+| &nbsp; | Property |
+|---|---|
+| Icon | `leading` on `ListTile` |
+
+##### Primary text
+
+| &nbsp; | Property |
+|---|---|
+| Text label | Not by default but can be made by using `ListTile()` and use property `title` |
+| Text color | Within `title` property you can customize the Text color by using the `style` property. The `style` property uses a `TextStyle` property and within that class use the `color` property |
+| Typography | Within `title` property you can customize the typography by using the `style property`. The `style` property uses a `TextStyle` property and within that class use the `fontFamily` property |
+
+##### Seconary text
+
+| &nbsp; | Property |
+|---|---|
+| Text label | Not by default but can be made by using `ListTile()` and use property `subtitle` |
+| Text color | Within `subtitle` property you can customize the Text color by using the `style` property. The `style` property uses a `TextStyle` property and within that class use the `color` property |
+| Typography | Within `subtitle` property you can customize the typography by using the `style property`. The `style` property uses a `TextStyle` property and within that class use the `fontFamily` property |
+
+
+##### Tertiary text
+
+| &nbsp; | Property |
+|---|---|
+| Text label | Not by default but can be made by using `ListTile()` and use property `subtitle` and enabling `isThreeLine` |
+| Text color | Within `subtitle` property you can customize the Text color by using the `style` property. The `style` property uses a `TextStyle` property and within that class use the `color` property |
+| Typography | Within `subtitle` property you can customize the typography by using the `style property`. The `style` property uses a `TextStyle` property and within that class use the `fontFamily` property |
+
+
+##### Metadata
+
+| &nbsp; | Property |
+|---|---|
+| Text label | Not by default but can be made by using `ListTile()` and use property `trailing` |
+| Text color | Within `trailing` property you can customize the Text color by using the `style` property. The `style` property uses a `TextStyle` property and within that class use the `color` property |
+| Typography | Within `trailing` property you can customize the typography by using the `style property`. The `style` property uses a `TextStyle` property and within that class use the `fontFamily` property |
+
+
+##### Controls
+
+| &nbsp; | Property |
+|---|---|
+| Primary action | Add interactive widget to `leading` on `ListTile` |
+| Secondary action | Add interactive widget to `trailing` on `ListTile` |
+
+### Theming lists
+
+Lists support [Material Theming](https://material.io/components/buttons/#theming) and can be
+customized in terms of color, typography and shape.
```

Comment:
Shape isn't really customizable for lists I think

Reply:
True removed.